src/eradius_proxy.erl

```diff
@@ -66,13 +77,21 @@ validate_arguments(Args) ->
             {true, [{default_route, DefaultRoute}, {options, Options}, {routes, NewRoutes}]}
     end.
 
-compile_routes(Routes) ->
-    RoutesOpts = lists:map(fun ({Name, Relay}) ->
+compile_routes(false) ->
+    [];
+compile_routes({routes, Routes}) ->
+    RoutesOpts = lists:map(fun (Route) ->
+                          {Name, Relay, Pool} = case Route of
```

Comment:
looks like you need align left `{Name, Relay, Pool}` on a level with `case re:compile(Name) of`

Reply:
fixed